vimeo-networking/src/main/java/com/vimeo/networking/model/VimeoAccount.java

```diff
@@ -39,81 +40,156 @@
 public class VimeoAccount implements Serializable {
 
     private static final long serialVersionUID = -8341071767843490585L;
-    //    private static final String TOKEN_TYPE_BEARER = "bearer";
 
+    @Nullable
     @GsonAdapterKey("access_token")
-    public String accessToken;
+    public String mAccessToken;
+
+    @Nullable
     @GsonAdapterKey("token_type")
-    public String tokenType;
+    public String mTokenType;
+
+    @Nullable
     @GsonAdapterKey("scope")
-    public String scope;
+    public String mScope;
+
+    @Nullable
     @GsonAdapterKey("user")
-    public User user;
-    private String userJSON;
+    public User mUser;
+
+    @Nullable
+    private String mUserJson;
 
-    public VimeoAccount(){
+    public VimeoAccount() {
         //constructor for stag TypeAdapter generation
     }
 
-    public VimeoAccount(@Nullable String accessToken) {
-        this.accessToken = accessToken;
+    /**
+     * Constructor for use for developers making requests with only one account. Provide your access
+     * token to the Configuration Builder and this constructor will be used to set up your authentication.
+     * <p>
+     * See the README of this library for more information.
+     *
+     * @param accessToken your access token.
+     * @see "https://github.com/vimeo/vimeo-networking-java#initialization"
+     */
+    public VimeoAccount(@NotNull String accessToken) {
+        //noinspection ConstantConditions
+        if (accessToken == null || accessToken.isEmpty()) {
+            throw new AssertionError("Account must be created with token");
+        }
+
+        mAccessToken = accessToken;
     }
 
-    public VimeoAccount(String accessToken, String tokenType, String scope, String userJSON) {
-        if (accessToken == null || accessToken.isEmpty() || tokenType == null ||
-            tokenType.isEmpty() || scope == null || scope.isEmpty()) {
+    /**
+     * Using this constructor is to create an account manually. It is recommended that if you cannot obtain
```

Comment:
Maybe it is better worded as: "Use this constructor to create an account manually. If you cannot obtain a valid access token, it is recommended to have a VimeoAccount constructed automatically using one of the authentication methods in VimeoClient rather than calling this constructor directly." If I am misunderstanding the docs, please clarify.